README.md

```diff
@@ -1,4 +1,8 @@
 # eztz - Javascript API library for Tezos
+
+[![Build
+Status](https://travis-ci.org/prashantpawar/eztz.svg?branch=master)](https://travis-ci.org/prashantpawar/eztz)
```

Comment:
You wanna change this to pointing to your own Travis-CI instance and repo (because otherwise, it'd be showing the status of my fork).